docs/tutorials/1_train_bc.ipynb

```diff
@@ -3,6 +3,7 @@
   {
    "cell_type": "markdown",
    "metadata": {},
+   "outputs": [],
```

Comment:
Lots of changes like this, was that intended/desired? If not you may want to revert this and other notebooks.

Reply:
Not intended, this should not be the case, thanks for spotting it.